src/components/navbar/Navbar.jsx

```diff
@@ -25,113 +25,64 @@ const navbarLinks = [
 
 const Navbar = () => {
   const [isNavOpen, setIsNavOpen] = useState(false);
-  const [width, setWidth] = useState(window.innerWidth);
-  useEffect(() => {
-    const handleResize = () => {
-      setWidth(window.innerWidth);
-    };
-
-    // Add a resize event listener
-    window.addEventListener("resize", handleResize);
-
-    // Clean up the event listener when the component unmounts
-    return () => {
-      window.removeEventListener("resize", handleResize);
-    };
-  }, []);
   const toggleNav = () => {
     setIsNavOpen(!isNavOpen);
   };
-  if (width > 700) {
-    return (
-      <nav
-        className="flex flex-row justify-between p-3"
-        style={{
-          backgroundColor: "#121930",
-          padding: "10px 60px",
-          alignItems: "center",
-          position: "sticky",
-          top: 0,
-          zIndex: 100,
-        }}
+
+  return (
+    <nav
+      className="flex flex-row justify-between p-3"
+      style={{
+        backgroundColor: "#121930",
+        padding: "10px 60px",
+        alignItems: "center",
```

Comment:
Please remove the styles which are present in tailwindcss, and add the corresponding tailwindcss class instead.